template/utils/send_sample_request.js

```diff
@@ -134,6 +135,11 @@ define([
             error      : displayError
         };
 
+        if(header['Content-Type'] == 'multipart/form-data'){
+            delete ajaxRequest.headers['Content-Type'];
+            ajaxRequest.contentType=false;
```

Comment:
why do you delete the Content-Type header?

Reply:
because jquery will automatic add Content-Type with bundry info when post formdata .  if don't delete    Content-Type header , the server side can't recognize correctly bundry info.